src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -186,14 +186,20 @@ Supported in Sentry Android Gradle Plugin version `3.0.0` and above.
 
 </Note>
 
-The [Sentry Android Gradle Plugin](/platforms/android/proguard/#gradle-configuration) does the tracing auto instrumentation via bytecode manipulation for `SQLite` and `Room` libraries.
+The [Sentry Android Gradle Plugin](/platforms/android/proguard/#gradle-configuration) does the tracing auto instrumentation using bytecode manipulation for `androidx.sqlite` and `androidx.room` libraries.
 
 The Plugin injects a code snippet that starts a span out of the active span bound to the scope for each `CRUD` operation. The SDK sets the span `operation` to `db` and `description` to the SQL Query if available.
 
 The span finishes once the operation has been executed. The span `status` is set to `SpanStatus.OK` if successful or `SpanStatus.INTERNAL_ERROR` if there was any error.
 
 When the operation throws an `Exception`, Sentry's SDK associates this exception to the running span. If you haven't set the SDK to swallow the exception and capture it, the span and SentryEvent will be linked when viewing it on the **Issue Details** page in sentry.io.
 
+<Note>
+ 
+At the moment, we only support standard `androidx.room` usage. For example, SQL queries for any `SupportSQLiteOpenHelper.Factory` other than [androidx.sqlite](https://github.com/androidx/androidx/tree/androidx-main/sqlite) will not be reported. However, it's worth reporting an [issue](https://github.com/getsentry/sentry-android-gradle-plugin/issues), as we might consider adding a new integration.
```

Comment:
Okay how about this (also, apologies, I replaced i.e., with "for example" instead of "in other words/that is"). 

```suggestion
At the moment, we only support standard `androidx.room` usage. That is, the SDK will not report any SQL queries for any `SupportSQLiteOpenHelper.Factory` other than [androidx.sqlite](https://github.com/androidx/androidx/tree/androidx-main/sqlite). However, if you are using a different `SupportSQLiteOpenHelper.Factory`, please report an [issue on Github](https://github.com/getsentry/sentry-android-gradle-plugin/issues), so we are aware and can possibly work on it.
```

Reply:
that looks perfect, thanks @imatwawana

Reply:
And thanks @philipphofmann for pointing out